components/header-bar/package.json

```diff
@@ -26,7 +26,7 @@
         "test": "d2-app-scripts test --jestConfig ../../jest.config.shared.js"
     },
     "peerDependencies": {
-        "@dhis2/app-runtime": "^2.11.0",
+        "@dhis2/app-runtime": "^2.12.1",
```

Comment:
You need to update this in every package to be able to resolve to a single version of it.

Tip: run `yarn manypkg check` and `yarn manypkg fix`in the UI root.